File: app/src/main/java/ie/dublinbuspal/android/data/remote/soap/xml/route/RouteServiceInboundStopXml.java
package ie.dublinbuspal.android.data.remote.soap.xml.route;

import org.simpleframework.xml.Element;
import org.simpleframework.xml.Root;

@Root(name = "InboundStop", strict = false)
public class RouteServiceInboundStopXml {

    @Element(name = "StopNumber", required = false)
    private String stopId;

    public RouteServiceInboundStopXml() {
        super();
    }

    public String getStopId() {
        return stopId;
    }

    public void setStopId(String stopId) {
        this.stopId = stopId;
    }

}

File: service/src/main/kotlin/ie/dublinbuspal/service/model/routeservice/RouteServiceInboundStopXml.kt
package ie.dublinbuspal.service.model.routeservice

import org.simpleframework.xml.Element
import org.simpleframework.xml.Root

@Root(name = "InboundStop", strict = false)
data class RouteServiceInboundStopXml(
        @field:Element(name = "StopNumber", required = false) var id: String? = null
)
